go.mod

```diff
@@ -1,10 +1,10 @@
 module github.com/chaos-mesh/chaos-mesh
 
-go 1.20
+go 1.22.5
```

Comment:
Use `1.22.5` because the latest version of github.com/99designs/gqlgen only supports go `1.22.5+`.